app/lib/common/utils/utils.dart

```diff
@@ -70,8 +71,27 @@ extension RefDebounceExtension on Ref {
 
 const largeScreenBreakPoint = 770;
 
-bool isLargeScreen(BuildContext context) {
-  return MediaQuery.of(context).size.width >= largeScreenBreakPoint;
+extension ActerContextUtils on BuildContext {
+  bool get isLargeScreen =>
+      MediaQuery.of(this).size.width >= largeScreenBreakPoint;
+
+  NavigatorState closeDialog({Routes? orGo, bool ignorePopFailure = false}) {
```

Comment:
Won't be `andGo`?
Mean close dialog and go to specific route?

Reply:
No. it means close the dialog if possible, OR if there is nothing to pop go there.

Reply:
there is at least one case where from the context of the current code it isn't clear if that can be issues without there being a place to `pop` to and thus it does `context.canPop() ? context.pop() : context.goNamed(...` . `orGo` This covers that pattern.